app/controllers/workflows_controller.rb

```diff
@@ -14,20 +14,17 @@ class WorkflowsController < ApplicationController
 
```

Comment:
Associated with unused tasks. Corresponding tasks have all been removed from the database.